README.md

```diff
@@ -1,7 +1,7 @@
-Akeneo Spreadsheet Parser
+LumenAd Spreadsheet Parser
 =========================
 
-This component is designed to extract data from spreadsheets, while being easy on resources, even for large files.
+This component is derivative of akeneo-labs (<insert link here>) and designed to extract data from spreadsheets, while being easy on resources, even for large files.
```

Comment:
Can we add the github link to akeneo-labs?

Reply:
> Can we add the github link to akeneo-labs?

Added